workspace.json

```diff
@@ -418,8 +418,9 @@
     "@fluentui/react-conformance": {
       "root": "packages/react-conformance",
       "projectType": "library",
+      "sourceRoot": "packages/react-conformance-griffel/src",
```

Comment:
Should be point to `react-conformance` instead of `react-conformance-griffel`?
```suggestion
      "sourceRoot": "packages/react-conformance/src",
```

Reply:
ofc, good catch!